_extras/data.md

```diff
@@ -0,0 +1,75 @@
+management, but with many complexities removed to allow students to focus on the core ideas and skills being taught.  
+ 
+ | Dataset | File name | Description |
+ | ---- | ------| ---- | 
+ | Site layout shapefiles | NEON-DS-Site-Layout-Files.zip | A set of shapefiles for the NEON's Harvard Forest field site and US and (some) state boundary layers. | 
+ | Meteorological data |  NEON-DS-Met-Time-Series.zip | Precipitation, temperature and other variables collected from a flux tower at the NEON Harvard Forest site
+ | Airborne remote sensing data | NEON-DS-Airborne-RemoteSensing.zip | LiDAR data collected by the NEON Airborne Observation Platform (AOP) and processed at NEON including a canopy height model, digital elevation model and digital surface model for NEON's Harvard Forest and San Joaquin Experimental Range field sites. | 
+ | Landstat 7 NDVI raster data | NEON-DS-Landsat-NDVI.zip | 2011 NDVI data product derived from Landsat 7 and processed by USGS cropped to NEON's Harvard Forest and San Joaquin Experimental Range field sites | 
+
+## About the Site Layout Shapefiles
+These vector data provide information on the site characterization and infrastructure at the 
+[National Ecological Observatory Network's](https://www.neonscience.org/)
+[Harvard Forest](https://www.neonscience.org/field-sites/field-sites-map/HARV) field site. 
+The Harvard Forest shapefiles are from the [Harvard Forest GIS & Map](http://harvardforest.fas.harvard.edu/gis-maps/) archives. 
+US Country and State Boundary layers are from the [US Census Bureau](https://www.census.gov/geo/maps-data/data/tiger-cart-boundary.html).
+
+## About the Meteorological Data
+The data used in this lesson were collected at the [National Ecological Observatory Network's](https://www.neonscience.org/) 
+[Harvard Forest](https://www.neonscience.org/field-sites/field-sites-map/HARV). These data are proxy data for what will be available 
+for 30 years on the [NEON data portal](http://data.neonscience.org/home) for the Harvard Forest and other field sites located across
+the United States.
+
+## About the Airborne Remote Sensing Data
+This data was collected at the [National Ecological Observatory Network's](https://www.neonscience.org/) 
+[Harvard Forest](https://www.neonscience.org/field-sites/field-sites-map/HARV) and 
+[San Joaquin Experimental Range](https://www.neonscience.org/field-sites/field-sites-map/SJER) field sites. 
+
+Airborne remote sensing flight surveys are conducted by the NEON project to collect quantitative information on land cover
+and changes to ecological structure and chemistry, including the presence and effects of invasive species across landscapes 100-300km2
+surrounding NEON sites. Data are collected during peak greenness at each field site and include the following: 
+
+* DSM (Digital Surface Model; full mosaic)
+* DTM (Digital Terrain Model; full mosaic)
+* CHM (Canopy Height Model; full mosaic)
+* RGB imagery (Harvard Forest site only)
+
```

Comment:
@ErinBecker The list of items here is correct for the included data set but is not correct for what NEON collects, which I feel is what this paragraph is describing. NEON has >30 remote sensing data products.  Instead of listing them all, I'd change it to something like: 

* Discrete LiDAR data
* Waveform LiDAR data
* Hyperspectral remote sensing data (426 bands)
* RGB camera imagery data

Otherwise, re-write the paragraph to clearly highlight that the mentioned data products are the ones included here and not the extent of what NEON provides.

Reply:
I agree with that statement @mjones01 . rather than provide more information that is required for the workshop. how about:
`
Data are collected during peak greenness at each field site and are processed to provide useful data products to the community. The following NEON data products are used in these lessons:  
Data Created from Discrete Return (point clouds) Lidar Data: 
+* DSM (Digital Surface Model; full mosaic)
+* DTM (Digital Terrain Model; full mosaic)
+* CHM (Canopy Height Model; full mosaic)
and 
+* RGB imagery (Harvard Forest site only) derived from the RGB Camera
`
NOTE: i had a rgb image for SJER in there. i know i used it work for workshops @jsta is that layer no longer in the data? I kind of remember that we DIDN"T have data for harvard however. it's been a while tho.